library/Fission/Types.hs

```diff
@@ -478,8 +483,8 @@ instance App.Creator Fission where
           Right _   -> Right (appId, subdomain)
 
 instance App.Modifier Fission where
-  setCID userId url newCID copyFiles now = do
-    runDB (App.setCID userId url newCID copyFiles now) >>= \case
+  setCID userId url newCID size copyFiles now = do
```

Comment:
Let’s be super clear: `bytes`

Reply:
if the type is `Bytes` do we also want the variable named `Bytes`. I feel like `size: Bytes` is most clear

Reply:
Yup, great 👍